openassessment/templates/openassessmentblock/self/oa_self_assessment.html

```diff
@@ -31,25 +31,20 @@ <h2 class="step__title">
 
             {% block title %}
             <span class="step__status">
-              <span class="step__status__label">{% trans "This step's status" %}:</span>
               <span class="step__status__value">
                   <span class="copy">{% trans "In Progress" %}</span>
               </span>
             </span>
             {% endblock %}
-       </h2>
+       </h4>
     </header>
 
     {% block body %}
     <div class="ui-slidable__content" aria-labelledby="oa_self_{{ xblock_id }}" id="oa_self_{{ xblock_id }}_content">
         <div class="wrapper--step__content">
             <div class="step__content">
                 <article class="self-assessment__display">
-                    <header class="self-assessment__display__header">
-                        <h3 class="self-assessment__display__title">{% trans "Your Response" %}</h3>
-                    </header>
```

Comment:
I'm just having trouble figuring out where this is rendered, as I see "Your response" being in an h5 header.